Misc/NEWS.d/next/Library/2022-07-08-17-49-12.gh-issue-87822.F9dzkf.rst

```diff
@@ -0,0 +1 @@
+Make FrameSummary robust to unrepresentable values.
```

Comment:
```suggestion
When called with ``capture_locals=True``, the :mod:`traceback` module functions swallow exceptions raised from calls to ``repr()`` on local variables of frames. This is in order to prioritize the original exception over rendering errors.  An indication of the failure is printed in place of the missing value.
```

Reply:
Is it customary to include the author? I did not find any other message file that does this.

Reply:
> Is it customary to include the author

Someone includes, someone doesn't. `git grep "(Patch by "` on `Misc\NEWS.d` gives the following:

```plain
3.11.0b1.rst:for most cases. (Patch by Inada Naoki.)
3.5.0a1.rst:asyncio tests no longer rely on tests.txt file. (Patch by Vajrasky Kok)
3.5.0b1.rst:Property() docstrings are now writeable. (Patch by Berker Peksag.)
3.5.3rc1.rst:dictitems_contains no longer swallows compare errors. (Patch by Xiang Zhang)
3.6.0b2.rst:Enables Unicode for ps1/ps2 and input() prompts. (Patch by Eryk Sun)
3.6.0rc2.rst:completed. (Patch by HoHo-Ho)
3.6.1rc1.rst:Ignores blank lines in ._pth files (Patch by Alexey Izbyshev)
3.6.1rc1.rst:Implement :pep:`529` for io.FileIO (Patch by Eryk Sun)
3.6.1rc1.rst:winreg does not truncate string correctly (Patch by Eryk Sun)
3.6.6rc1.rst:``MetaPathFinder``. (Patch by Himanshu Lakhara)
3.7.0a1.rst:winreg does not truncate string correctly (Patch by Eryk Sun)
3.7.0a1.rst:Enables Unicode for ps1/ps2 and input() prompts. (Patch by Eryk Sun)
3.7.0a3.rst:argument to ``Traceback.format()``. (Patch by Jesse Bakker.)
3.7.0b3.rst:``MetaPathFinder``. (Patch by Himanshu Lakhara)
3.8.0a1.rst:``MetaPathFinder``. (Patch by Himanshu Lakhara)
3.9.0a1.rst:Make json.loads faster for long strings. (Patch by Marco Paolini)
3.9.0a1.rst:of :pep:`442`. Patch by Joannah Nanjekye. (Patch by Joannah Nanjekye)
3.9.0b1.rst:Initialize ``n->n_col_offset``. (Patch by Joannah Nanjekye)
```

Reply:
It’s standard to add the author in the change log, but not always done. Sometimes the release manager adds the name. If you do it you save them a few clicks.